cognite/replicator/files.py

```diff
@@ -0,0 +1,215 @@
+import logging
+import mimetypes
+import time
+from typing import Dict, List
+
+from cognite.client import CogniteClient
+from cognite.client.data_classes import FileMetadata
+from cognite.client.exceptions import CogniteAPIError
+
+from . import replication
+
+
+def create_file(
+    src_file: FileMetadata, src_dst_ids_assets: Dict[int, int], project_src: str, runtime: int
+) -> FileMetadata:
+    """
+    Makes a new copy of the file to be replicated based on a source file.
+
+    Args:
+        src_file: The file from the source to be replicated to destination.
+        src_dst_ids_assets: A dictionary of all the mappings of source asset id to destination asset id.
+        project_src: The name of the project the object is being replicated from.
+        runtime: The timestamp to be used in the new replicated metadata.
+
+    Returns:
+        The replicated file to be created in the destination.
+    """
+    logging.debug(f"Creating a new file based on source file id {src_file.id}")
+    mime_type = mimetypes.types_map.get(f".{src_file.mime_type}", src_file.mime_type)
```

Comment:
Why use format string here? Can't you use src_file.mime_type directly, or if you need it be string, use the str() method. :)

Reply:
We store "pdf" but mimetypes library needs ".pdf" - maybe str concat "." + str(...) would be more clear

Reply:
Ah, I missed the dot, then it is perfect :)